### ISSUE TITLE
Cleanup CNI documentation ("how-to-implement-the-kubernetes-networking-model")

### ISSUE THREAD

**Issue by jimangel** [kubernetes/website#22961]
**This is a Feature Request**

> discussed at the 8/4/2020 SIG Docs meeting

The page [how-to-implement-the-kubernetes-networking-model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model) is an eyesore and not very helpful to a Kubernetes newcomer.

In addition, some of the listed networking plugins blur the line of acceptable [3rd party content](https://github.com/kubernetes/enhancements/blob/master/keps/sig-docs/20191020-doc-policies-for-third-party-content.md). Some, but not all, networking providers fall under the criteria of "third-party OSS content [...] that [is required] to function."

The resolution consists of three parts:

1. Modify the page to be more accessible for folks new to Kubernetes with clear "next step" guidance.

1. Outline the specific criteria of what 3rd party content is accepted, and why, for **specific page(s)** in the [content-guide](https://kubernetes.io/docs/contribute/style/content-guide/). Mainly the criteria for accepting or rejecting content.

1. Remove any providers that do not meet specific outlined criteria

**Why is this needed**

A cluster provisioning tool `kubeadm` has stopped providing examples of CNI providers (dual sourced content) due to the fact that there has been an increase in "tag along" PRs. `kubeadm` does not want to be in the business of choosing "winners" for CNI's and directs users to [how-to-implement-the-kubernetes-networking-model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model) as a result. (reference PR: https://github.com/kubernetes/website/pull/22600)

[how-to-implement-the-kubernetes-networking-model](https://kubernetes.io/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model) needs to be cleaned up if we are sending more traffic there.

The content-guide update is needed for reference if SIG Docs closes a PR that does not meet the "third-party OSS content [...] that [is required] to function" criteria. In addition it could assist with decision making. (reference PR: https://github.com/kubernetes/website/pull/22799)

/cc @sftim @kbarnard10 @onlydole @kbhawkey @zacharysarah @celestehorgan

**Comment by sftim** [comment]
At least some details about cluster networking will find a better home inside https://kubernetes.io/docs/concepts/services-networking/

**Comment by sftim** [comment]
This is also more distantly relevant to the _other_ [SIG Network](https://github.com/cncf/sig-network).

**Comment by ckranz** [comment]
Agreed on the issue and challenges here. I made my first PR (22973) to try address the issue of the simple install instructions being removed from the kubeadm page. In doing so I reviewed all the projects to try see if they had simple install instructions so I could add them, but ended up very confused myself! Additionally, a few of the projects are only relevant if you're using the vendors specific cloud / infrastructure / platform, but that's not always clear from the description.

**Comment by onlydole** [comment]
/remove-lifecycle stale

Happy New Year, everyone!

**Comment by sftim** [comment]
I'd be happy to move the list of network plugins into the reference section.

**Comment by tengqm** [comment]
/close
I believe the section mentioned in this issue report has been revised long ago.
No 3rd party contents is still there.
